backend/src/test/resources/spring.properties

```diff
@@ -0,0 +1 @@
+spring.test.constructor.autowire.mode=all
```

Comment:
Súhlas s autowarningom - na konci config file mám rád newline (prázdny riadok).

Reply:
Done in [d4e080e](https://github.com/cesko-digital/loono-be/pull/31/commits/d4e080e47aa5594d43cdc57ffc7b2b4736f85f2c)